### ISSUE TITLE
A11y_SQLdatabase projects extension_Screen Reader: VoiceOver is not announcing incorrect role for 'read our docs' as button.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/azuredatastudio#22077]
### GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #MAS1.3.1; #MAC; #AINotInScope; #Info and Relationships; #Voiceover; #Element:Button.

### Environment Details:​
Application Name: SQL Azure Data Studio
PowerShell Verion: v2022.5.0
MAC Version: macOS Monterey 12.2.1

### Repro Steps:
Start Voiceover with using shortcut key Command+F5.
Launch Azure Data Studio
Tab till extension and search for “SQL Database Projects” and install it.
Tab till database project and press enter.
“Database Project” panel will open.
Tab till 'Read our docs'
Verify whether vioceover is announcing correct role for 'read our docs' as link or not.

### Actual Result:​
Voiceover is announcing incorrect role for 'read our docs' as button instead of link.
Visually 'read our docs' looks like link and also working as link but voiceover is announcing role as button.

### Expected Result:​
Voiceover should be announcing role for 'read our docs' as link instead of button.

### ![Presentation](https://user-images.githubusercontent.com/86241494/222136254-4e8ca45c-252a-437e-a12c-3e9896273574.gif)
Vioceover should be announce role for 'read our docs' as link.

**Comment by kisantia** [comment]
Upstream issue - filed an issue for this in VS Code: https://github.com/microsoft/vscode/issues/175812
